src/platforms/web/runtime/class-util.js

```diff
@@ -25,6 +32,11 @@ export function addClass (el: Element, cls: string) {
  * SVG elements in IE
  */
 export function removeClass (el: Element, cls: string) {
```

Comment:
You're right, I totally missed this. Adding it in, though maybe we don't need to cater for `null` or `undefined` at all (a "`.trim()` is not a function" error will be thrown).